packages/checkout-ui-extensions-run/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@shopify/checkout-ui-extensions-run",
+  "version": "0.4.1",
+  "description": "A command line interface for developing and building Checkout UI extensions",
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "bin": {
+    "checkout-ui-extensions-run": "./bin/checkout-ui-extensions-run"
```

Comment:
This is a very, very long name for a binary... Can we come up with anything else? Maybe it's not a huge deal if this CLI stuff is on the way out

Reply:
Ok, I'll keep the current name